tethys_services/backends/hs_restclient_helper.py

```diff
@@ -0,0 +1,29 @@
+import hs_restclient as hs_r
+from django.conf import settings
+
+
+def get_oauth_hs(request):
+    hs = None
+    error_msg_head = "Failed to initialize hs object: "
+
+    if request.user.social_auth.count() == 0:
+        raise Exception(error_msg_head + "Not logged in through OAuth")
+    if request.user.social_auth.count() > 1:
```

Comment:
I'm pretty sure it is perfectly legal for users to link multiple social accounts to their Tethys account, in which case the current handling of multiple social auth objects would create a block to using the HydroShare client. Can you change this to support multiple social auth accounts?